docs/source/nlp/tutorials/quickstarts/plot_multi_label_classification.py

```diff
@@ -0,0 +1,228 @@
+multi label classification task. We will cover the following:
+
+1. `Creating a TextData object and auto calculating properties <#setting-up>`__
+2. `Running the built-in suites <#running-the-deepchecks-default-suites>`__
+3. `Running individual checks <#running-individual-checks>`__
+
+To run deepchecks for NLP, you need the following for both your train and test data:
+
+1. Your text data - a list of strings, each string is a single sample (can be a sentence, paragraph, document, etc.).
+2. Your labels and prediction in the :ref:`correct format <nlp__supported_text_classification>` (Optional).
+3. :ref:`Metadata <nlp__metadata_guide>`, :ref:`Properties <nlp__properties_guide>`
+   or :ref:`Embeddings <nlp__embeddings_guide>` for the provided text data (Optional).
+
+If you don't have deepchecks installed yet:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install deepchecks[nlp] -U --quiet #--user
+
+Some properties calculated by ``deepchecks.nlp`` require additional packages to be installed. You can
+install them by running:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install [nlp-properties] -U --quiet #--user
+
+Setting Up
+==========
+
+Load Data
+---------
+For the purpose of this guide, we'll use a small subset of the
+`just dance <https://www.kaggle.com/datasets/renatojmsantos/just-dance-on-youtube>`__ comment analysis dataset.
+Dataset containing comments, metadata and labels for multilabel category classification of the comments.
+
+"""
+
+from deepchecks.nlp import TextData
+from deepchecks.nlp.datasets.classification import just_dance_comment_analysis
+
+data = just_dance_comment_analysis.load_data(data_format='DataFrame', as_train_test=False)
+metadata_cols = ['likes', 'dateComment']
+data.head(2)
+
+# %%
+# Create TextData Objects
+# ------------------------
+#
+# Deepchecks' :ref:`TextData <nlp__textdata_object>` object contains the text samples, labels, and possibly
+# also properties and metadata. It stores
+# cache to save time between repeated computations and contains functionalities for input validations and sampling.
+
+label_cols = data.drop(columns=['originalText'] + metadata_cols)
+class_names = label_cols.columns.to_list()
+dataset = TextData(data['originalText'], label=label_cols.to_numpy().astype(int),
+                   task_type='text_classification', metadata=data[metadata_cols], categorical_metadata=[])
+
+# %%
+# Calculating Properties
+# ----------------------
+#
+# Some of deepchecks' checks use properties of the text samples for various calculations. Deepcheck has a wide variety
+# of such properties, some simple and some that rely on external models and are more heavy to run. In order for
+# deepchecks' checks to be able to access the properties, they must be stored within the TextData object.
+
+# properties can be either calculated directly by Deepchecks or
+# imported from other sources in an :ref:`appropriate format <nlp__properties_guide>.
```

Comment:
Missing a much more "direct" mention & link to the properties guide (currently only referred to when mentioning the appropriate format)